lib/deepl.rb

```diff
@@ -38,6 +40,11 @@ def api
     @api ||= API.new(configuration)
   end
 
+  def glossary(name, entries, source_lang, target_lang, options = {})
+    configure if @configuration.nil?
+    Requests::Glossary.new(api, name, entries, source_lang, target_lang, options).request
```

Comment:
Since there are multiple resources related to glossaries, this should be named `Requests::Glossary::Create` or `Requests::CreateGlossary`. I think the first ones matches properly the restful namespacing.